pkgs/top-level/all-packages.nix

```diff
@@ -8334,6 +8334,12 @@ with pkgs;
   nodejs_latest = nodejs-18_x;
   nodejs-slim_latest = nodejs-slim-18_x;
 
+  buildNpmPackage = callPackage ../build-support/node/build-npm-package { };
+
+  inherit (callPackage ../build-support/node/fetch-npm-deps {
+    inherit (darwin.apple_sdk.frameworks) Security;
```

Comment:
Any chance we can use `apple_sdk_11`? IIRC an attempt is being made to use that as much as possible.

Reply:
There is no need for this, and I am not aware of any effort like that -- we should only use it when needed.